core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/CheBootstrap.java

```diff
@@ -239,11 +243,16 @@ protected void bindProperties(String prefix, Map<String, String> properties) {
                             if (actual == null) {
                                 actual = System.getenv(name);
                             }
-                            if (actual == null) {
+                            if (actual != null) {
```

Comment:
I think I will have +1 level nesting in this case. Can you provide full example?